ocs_ci/ocs/perftests.py

```diff
@@ -841,3 +850,32 @@ def check_results_and_push_to_dashboard(self):
             log.info(f"Check results for [{wl['name']}] in : {self.results_file}")
             self.check_tests_results()
             self.push_to_dashboard(test_name=wl["test_name"])
+
+    def create_test_project(self):
+        """
+        Creating new project (namespace) for performance test
+        """
+        self.namespace = "pas-test-namespace"
+        log.info(f"Creating new namespace ({self.namespace}) for the test")
+        try:
+            self.proj = helpers.create_project(project_name=self.namespace)
+        except CommandFailed as ex:
+            if str(ex).find("(AlreadyExists)"):
+                log.warning("The namespace is already exists !")
```

Comment:
Spelling fix: The namespace already exists!

Reply:
done